### PR TITLE
[14.0][MIG] delivery_line_sale_line_position

### PR DESCRIPTION
**Pull request by hailangvn**
Depends on:
* `sale_order_line_position`: https://github.com/OCA/sale-reporting/pull/118

### CODE REVIEW

File: delivery_line_sale_line_position/README.rst [hunk 1]
```diff
@@ -59,6 +59,12 @@ Contributors
 ~~~~~~~~~~~~
 
 * Thierry Ducrest <thierry.ducrest@camptocamp.com>
+* `Trobz <https://trobz.com>`_:
```
Comment (mmequignon) [inline delivery_line_sale_line_position/README.rst:62]: aren't you missing your name here ?

**Review by mmequignon — APPROVED** [review]

**Review by sebalix — APPROVED** [review]

**Review by simahawk — COMMENTED** [review]
LG base PR is getting merged

**Comment by simahawk** [comment]
@hailangvn can you drop the pending merge?

**Comment by hailangvn** [comment]
Thank @simahawk , this PR has been updated.

**Pull request merged**